app/konnect/runtime-manager/runtime-instances/index.md

```diff
@@ -4,14 +4,14 @@ content_type: explanation
 disable_image_expand: true
 ---
 
-A runtime instance is a single self-managed instance of {{site.base_gateway}} that functions as a data plane. In {{site.konnect_saas}}, runtime instances are part of [runtime groups](/konnect/runtime-manager/runtime-groups) and can be used to serve traffic to the runtime group. Runtime groups manage and store configuration in {{site.konnect_saas}}, and runtime instances are configured according to the configuration distributed by the runtime group.
+A data plane node is a single self-managed instance of {{site.base_gateway}} that functions as a data plane. In {{site.konnect_saas}}, data plane nodes are part of [control planes](/konnect/runtime-manager/runtime-groups) and can be used to serve traffic to the control plane. Control planes manage and store configurations in {{site.konnect_saas}}, and data plane nodes are configured according to the configuration distributed by the control plane.
```

Comment:
```suggestion
A data plane node is a single self-managed instance of {{site.base_gateway}} that acts as a proxy and serves traffic. 
In {{site.konnect_saas}}, data plane nodes are managed by [control planes](/konnect/runtime-manager/runtime-groups/). 
Control planes manage and store configurations in {{site.konnect_saas}}, and data plane nodes are configured according to the configuration distributed by the control plane.
```
It is much clearer now how much of this paragraph was just gibberish when we were trying to use the old terms.